Sources/sourcekit-lsp/SourceKitLSP.swift

```diff
@@ -262,35 +278,35 @@ struct SourceKitLSP: AsyncParsableCommand {
     )
     cleanOldLogFiles(logFileDirectory: logFileDirectoryURL, maxAge: 60 * 60 /* 1h */)
 
+    let inputMirror = orLog("Setting up input mirror") {
+      if let inputMirrorDirectory = globalConfigurationOptions.loggingOrDefault.inputMirrorDirectory {
```

Comment:
Might be worth documenting that these options are global only, not per project

Reply:
We have documentation on the `logging` configuration key that these options are global only already.

Reply:
So we do, thanks 👍